src/utils.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ *  Copyright 2021 EPAM Systems
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+import path from 'path';
+import fs from 'fs';
+import { ReportPortalConfig } from './models';
+import { RP_CONFIG_FILE_NAME } from './constants';
+
+export const getConfig = (providedConfig?: ReportPortalConfig): ReportPortalConfig => {
+  try {
+    if (!providedConfig || Object.keys(providedConfig).length === 0) {
+      return JSON.parse(fs.readFileSync(path.resolve(RP_CONFIG_FILE_NAME)).toString());
```

Comment:
What is the benefit using `json` and read it info against provide config to the `playwright.config.ts` and grab it in `onBegin` method?
[https://playwright.dev/docs/test-reporters#multiple-reporters](https://playwright.dev/docs/test-reporters#multiple-reporters)

Reply:
Yes, we can provide config with `playwright.config.ts` and `globalSetup`, and then we will have to move 
` this.RPconfig = JSON.parse(process.env.RPconfig)
    this.client = new RPClient(this.RPconfig);`
 from `constructor(config)` to `onBegin` method.
 But what to do with tests, which using `constuctor(config)` for creating instance class. How provide config and testing this case?
 I think it's better to use the `getConfig` util and provide config via constructor, but maybe I'm wrong.

Reply:
Please, can you provide short example test which needed this additional config?
Also, as I see your reporter class don't takes params.